### PR TITLE
Feature/m0081 meja 1648 architecture 7 17 0

### PR DESCRIPTION
**Pull request by jerome-leroux**

### CODE REVIEW

File: PlatformDeveloperGuide/architectureChangelog.rst [hunk 1]
```diff
@@ -32,13 +32,69 @@ specific configuration:
...
+
+-  Updated ``KF`` to version ``1.6``:
+  
+   -  Added ``Kernel.canUninstall()`` method.
+
+Integration
+~~~~~~~~~~~
+
+-  Fixed Architecture compatibility issue with Java 11.
+-  Fixed missing default value for ShieldedPlug server port when running it with MMM.
```
Comment (frivieremicroej) [inline PlatformDeveloperGuide/architectureChangelog.rst:62]: can you put the default value 
```suggestion
-  Fixed missing default value for ShieldedPlug server port when running it with MMM (``xxx``).
```

File: PlatformDeveloperGuide/architectureChangelog.rst [hunk 2]
```diff
@@ -32,13 +32,69 @@ specific configuration:
...
+-  Updated Memory Map Scripts for ``com.microej.api.vectorgraphics`` library.
+-  Updated Architecture End User License Agreement to version ``SDK 3.1-A``.
+
+Simulator
+~~~~~~~~~
+
+-  Added class file major version check (<=51). Classes mut be compiler for Java 7 or lower. Set the options property ``S3.DisableClassFileVersionCheck`` to ``false`` to disable this verification.
```
Comment (frivieremicroej) [inline PlatformDeveloperGuide/architectureChangelog.rst:69]:
```suggestion
-  Added class file major version check (<=51). Classes must be compiled for Java 7 or lower. Set the options property ``S3.DisableClassFileVersionCheck`` to ``false`` to disable this verification.
```

File: PlatformDeveloperGuide/architectureChangelog.rst [hunk 3]
```diff
@@ -32,13 +32,69 @@ specific configuration:
...
+-  Added native method signature in the stack trace of the ``UnsatisfiedLinkError`` thrown when a native method is missing.
```
Comment (frivieremicroej) [inline PlatformDeveloperGuide/architectureChangelog.rst:70]: use javadoc links (make a whole pass on changelog page ) https://youtrack.cross/issue/M0078DDG-316

File: PlatformDeveloperGuide/architectureChangelog.rst [hunk 4]
```diff
@@ -32,13 +32,69 @@ specific configuration:
...
+-  [Multi] - Fixed unexpected error when ``kernel.kf`` file is missing and KF library is used: "Please specify a 'kernel.kf' file to enable Kernel & Features semantics."
+-  [Multi] - Fixed type ``double[]`` not recognized in ``kernel.api`` file.
+
+SOAR
+~~~~
+
+-  Fixed internal error when using a BON constant in an if statement at the end of a ``try`` block.
+-  Fixed internal error when a ``try`` block ends with an ``assert`` expression while assertions are disabled.
+-  [Multi] - Raise a warning instead of an error when duplicated ``.kf`` files are detected in the Kernel classpath. Usual classpath resolution order is used to load the file (see :ref:`chapter.microej.classpath`).
+-  [Multi] - Fixed SOAR error when building a Feature that uses an array of basetypes that is not explicitly declared in the ``kernle.api`` file of the Kernel.
```
Comment (frivieremicroej) [inline PlatformDeveloperGuide/architectureChangelog.rst:87]:
```suggestion
-  [Multi] - Fixed SOAR error when building a Feature that uses an array of basetypes that is not explicitly declared in the ``kernel.api`` file of the Kernel.
```

File: PlatformDeveloperGuide/architectureChangelog.rst [hunk 5]
```diff
@@ -32,13 +32,69 @@ specific configuration:
...
+-  Updated ``KF`` to version ``1.6``:
+  
+   -  Added ``Kernel.canUninstall()`` method.
+
+Integration
+~~~~~~~~~~~
+
+-  Fixed Architecture compatibility issue with Java 11.
+-  Fixed missing default value for ShieldedPlug server port when running it with MMM.
+-  Updated Memory Map Scripts for ``com.microej.api.vectorgraphics`` library.
```
[the remark this answers is not in the document]
Comment (jerome-leroux) [reply PlatformDeveloperGuide/architectureChangelog.rst]: fixed

File: PlatformDeveloperGuide/architectureChangelog.rst [hunk 6]
```diff
@@ -32,13 +32,69 @@ specific configuration:
...
+~~~~~~~~~~~~~~~~~~~~
+
+-  Updated ``KF`` to version ``1.6``:
+  
+   -  Added ``Kernel.canUninstall()`` method.
+
+Integration
+~~~~~~~~~~~
+
+-  Fixed Architecture compatibility issue with Java 11.
```
[the remark this answers is not in the document]
Comment (jerome-leroux) [reply PlatformDeveloperGuide/architectureChangelog.rst]: New sentence proposed

File: PlatformDeveloperGuide/architectureChangelog.rst (hunk 3 above)
Comment (jerome-leroux) [reply PlatformDeveloperGuide/architectureChangelog.rst]: All the references to the classes and methods have been converted to links.

File: PlatformDeveloperGuide/architectureChangelog.rst (hunk 1 above)
Comment (jerome-leroux) [reply PlatformDeveloperGuide/architectureChangelog.rst]: fixed

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**